ui/src/main/js/page/settings/standalone/SettingsConfigurationStandalone.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import * as PropTypes from 'prop-types';
+import { Tab, Tabs } from 'react-bootstrap';
+import * as FieldModelUtilities from 'common/util/fieldModelUtilities';
+import { CONTEXT_TYPE } from 'common/util/descriptorUtilities';
+import CommonGlobalConfiguration from 'common/global/CommonGlobalConfiguration';
+import { SETTINGS_INFO } from 'page/settings/SettingsModel';
+import SettingsEncryptionConfiguration from 'page/settings/standalone/SettingsEncryptionConfiguration.js';
+
+const SettingsConfigurationStandalone = ({
+    csrfToken, errorHandler, readOnly, displaySave
+}) => {
+    const [formData, setFormData] = useState(FieldModelUtilities.createEmptyFieldModel([], CONTEXT_TYPE.GLOBAL, SETTINGS_INFO.key));
```

Comment:
I don't think you need this state, especially since it's using the old field model stuff. lastUpdated can be empty

Reply:
Paulo and I discussed this one and weren't entirely sure if we wanted it here or not. We think we may need to add it back in when we do the Proxy configuration later to get the time updated on the page. We left it as a reminder but I can remove it and find a solution to it in the Proxy UI config